Packs/CommunityCommonScripts/Scripts/GetListDatawithKeyword/GetListDatawithKeyword.py

```diff
@@ -0,0 +1,51 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+
+from typing import Dict, Any
+import traceback
+
+result_data=[]
+
+def get_data(key_word, json_data):
+    for i in range(len(json_data)):
+        for key,value in json_data[i].items():
+            if key_word in value:
+                result_data.append(json_data[i])
+                break
+            else:
+                pass
+
+    return result_data
+
+
+''' MAIN FUNCTION '''
+
+
+def main():
+    try:
+        key_word = demisto.args()['Keyword']
+        json_data = argToList(demisto.args()['value'])
```

Comment:
You are using the `value` as a required argument. but it does not included in the yml file of the integration